tests/helpers/testhelper.js

```diff
@@ -129,31 +129,6 @@ exports.setup.testRun = function (test, name) {
           return def.message === er.message;
         });
       });
-
-      test.ok(
-        undefinedErrors.length === 0 &&
-          unthrownErrors.length === 0 &&
-          wrongLineNumbers.length === 0 &&
-          duplicateErrors.length === 0,
```

Comment:
Hi @jugglinmike. I have made all the changes as you suggested except the last one, that is disabling unit test suite just for W041. I'm kind of stuck here. I'm thinking, I need to modify the code between line numbers 133 - 156 in testhelper.js. Tell me if I'm going in the right direction. Any help is appreciated.